tests/subsys/pm/power_residency_time/pytest/current_analyzer.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2024 Intel Corporation.
```

Comment:
would this be better to move this function to scripts/pm/pm_helper.py, and rms is not the only implement, e.g. we can have average mean